package.json

```diff
@@ -9,18 +9,20 @@
     "react": "^16.9.0",
     "react-dom": "^16.9.0",
     "react-router-dom": "^5.0.1",
-    "react-scripts": "3.0.1",
+    "react-scripts": "3.1.1",
     "react-table": "next",
     "react-toastify": "^5.3.2",
+    "semantic-ui-css": "^2.4.1",
+    "semantic-ui-react": "^0.88.0",
     "styled-components": "^4.3.2",
     "uuid": "^3.3.3",
     "yup": "^0.27.0"
   },
   "scripts": {
-    "start": "REACT_APP_API_URL=http://localhost:9000 craco start",
-    "build": "craco build",
-    "test": "craco test",
-    "eject": "craco eject"
+    "start": "REACT_APP_API_URL=http://localhost:9000 react-scripts start",
```

Comment:
After the latest discussion about using `REACT_APP_API_URL` env variable, we've decided to remove it from the script. Please delete it here as well 🙏